src/test/java/com/patreon/PatreonOAuthTest.java

```diff
@@ -0,0 +1,26 @@
+package com.patreon;
```

Comment:
is this file incomplete?

Reply:
yes, this and the other test are both placeholders